include/vkd3d_device_vkd3d_ext.idl

```diff
@@ -54,3 +54,18 @@ interface ID3D12DXVKInteropDevice : IUnknown
     HRESULT LockCommandQueue(ID3D12CommandQueue *queue);
     HRESULT UnlockCommandQueue(ID3D12CommandQueue *queue);
 }
+
+[
+    uuid(f3112584-41f9-348d-a59b-00b7e1d285d6),
+    object,
+    local,
+    pointer_default(unique)
+]
+interface ID3DLowLatencyDevice : IUnknown
```

Comment:
Is there a reason this is not augmenting the existing device extension interface?

Reply:
I guess this is to let dxvk-nvapi use the same common interface that can be queried from both vkd3d-proton's `ID3D12Device` and DXVK's `ID3D11Device`.

Reply:
Saancreed is correct. I did it this way to prevent having to query if the device provided to dxvk-nvapi is a D3D12 or D3D11 device.

Reply:
Ok, if DXVK and vkd3d-proton share the same iface, this is okay I guess.